include/SH3/arc/vfile.hpp

```diff
@@ -53,9 +49,11 @@ struct sh3_arc_vfile final
      *  @param mft      The @ref sh3_arc Master File Table, arc.arc.
      *  @param fname    The name of the file we want to open.
      *
+     *  @note If the file is already open, this function does nothing but returns @ref open.
```

Comment:
Shouldn't it perhaps return `false` because a different file could be open?
Actually, this method maybe should be private, since the Constructor always already opens the vfile. That way it should never happen that `Open()` is called twice.